Source/WebCore/editing/MarkupAccumulator.cpp

```diff
@@ -208,6 +209,17 @@ String MarkupAccumulator::serializeNodes(Node& targetNode, SerializedNodes root,
     return m_markup.toString();
 }
 
+bool MarkupAccumulator::appendContentsForNode(StringBuilder& result, const Node& targetNode)
+{
+    if (m_replacementURLStrings.isEmpty() || !targetNode.hasTagName(styleTag))
+        return false;
+
+    auto& styleElement = downcast<HTMLStyleElement>(targetNode);
+    result.append(styleElement.contentsStringWithReplacementURLs(m_replacementURLStrings));
```

Comment:
I'd call this `textContentWithReplacementURLs` or `stringWithReplacementURLs` instead.